android/canonical/app/src/main/java/com/google/samples/quickstart/canonical/RunFragment.kt

```diff
@@ -51,6 +50,8 @@ class RunFragment : Fragment() {
             param2 = it.getString(ARG_PARAM2)
         }
 
+//        stopwatchVM = StopwatchViewModel()
+
```

Comment:
Ok to just delete this code instead of commenting it out. We can look at a previous commit if we need it back.